gem/tests/test_auth.py

```diff
@@ -441,3 +447,291 @@ def test_auth_middleware(self):
     #     self.assertContains(response, "Welcome to the GEM Wagtail CMS")
     #     # test that the correct logout button is in in the template
     #     self.assertContains(response, "oidc/logout")
+
+
+class TestAllAuth(GemTestCaseMixin, TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_superuser(
+            username='superuser', email='superuser@email.com', password='pass')
+        self.main = self.mk_main(
+            title='main1', slug='main1',
+            path='00010002', url_path='/main1/'
+        )
+        self.site = self.main.get_site()
+
+    @override_settings(ENABLE_ALL_AUTH=True)
+    def test_admin_login_view(self):
+        res = self.client.get(reverse('wagtailadmin_login'))
+        self.assertEqual(res.status_code, 200)
+        self.assertEqual(settings.ENABLE_ALL_AUTH, True)
```

Comment:
Since you set this setting before the test you probably don't need to check it here.
What is this test actually checking? that the login view doesn't redirect? Is the user already authenticated?
Maybe you should be checking which url we land at or which template is used?